test/onnx/test_pytorch_onnx_no_runtime.py

```diff
@@ -775,6 +775,39 @@ def forward(self, x):
             model, inputs, f, dynamic_axes={"x": [0, 1]}, input_names=["x"]
         )
 
+    def test_col2im(self):
+        # This test can be moved to test/onnx/test_pytorch_onnx_onnxruntime.py when ORT implement ::Col2Im
```

Comment:
Can you create a tracking issue for this? Otherwise it risks being forgotten about.

Reply:
Tracking through https://github.com/pytorch/pytorch/issues/84408